### PR TITLE
upgrade aws sdk to v2

### PR DESCRIPTION
**Pull request by pmenglund**

### CODE REVIEW

**Comment by pmenglund** [comment]
Current dependencies on/for this PR:
* master
  * **PR #10** <a href="https://app.graphite.dev/github/pr/rockset/rockbench/10" target="_blank">Graphite</a> 
    * **PR #11** <a href="https://app.graphite.dev/github/pr/rockset/rockbench/11" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/rockset/rockbench/11?utm_source=stack-comment).

File: generator/snowflake.go [hunk 1]
```diff
@@ -152,7 +147,7 @@ func (r *Snowflake) ConfigureDestination() error {
 
 	// create stage
 	stageName := "perfstage" + r.generatorIdentifier
-	createStageQuery := "create stage " + stageName + " url='s3://" + r.stageS3BucketName + "' credentials = (AWS_KEY_ID = '" + credValue.AccessKeyID + "' AWS_SECRET_KEY = '" + credValue.SecretAccessKey + "' );"
+	createStageQuery := "create stage " + stageName + " url='s3://" + r.stageS3BucketName + "' credentials = (AWS_KEY_ID = '" + creds.AccessKeyID + "' AWS_SECRET_KEY = '" + creds.SecretAccessKey + "' );"
```
Comment (kwadhwa18) [inline generator/snowflake.go:150]: curious - since we are concatenating strings with secrets here, is this a good practice?

**Review by kwadhwa18 — APPROVED** [review]
nice!

File: generator/snowflake.go (hunk 1 above)
Comment (pmenglund) [reply generator/snowflake.go:150]: I don't know why we're doing this, is it a requirement to have them in the stage?

Comment (kwadhwa18) [reply generator/snowflake.go:150]: yes - snowflake takes in the credentials as part of this command https://docs.snowflake.com/en/sql-reference/sql/create-stage.html

**Pull request merged**